lib/Doctrine/MongoDB/CursorInterface.php

```diff
@@ -253,4 +253,13 @@ public function tailable($tail = true);
      * @return self
      */
     public function timeout($ms);
+
+    /**
+     * Wrapper method for MongoCursor::maxTimeMS().
+     *
+     * @see http://php.net/manual/en/mongocursor.maxtimems.php
+     * @param integer $ms
+     * @return self
+     */
+    public function maxTimeMS($ms);
```

Comment:
@ne0h12: Unfortunately, this needs to be removed. We cannot add new interface methods without bumping a major version.